src/FilePaths.jl

```diff
@@ -66,6 +68,7 @@ Base.next(p::AbstractPath, i::Int) = next(String(p), i)
 Base.String(path::AbstractPath) = error("`String not implemented")
 parts(path::AbstractPath) = error("`parts` not implemented.")
 root(path::AbstractPath) = error("`root` not implemented.")
+drive(path::AbstractPath) = error("`drive` not implemented.")
 
```

Comment:
Should this default to returning an empty string? https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.drive

Reply:
I mainly tried to follow the convention for the other public API functions here that all had one method that throws an error for ``AbstractPath``, and then two separate methods that work for real, one for ``PosixPath`` and one for ``WindowsPath``. So there is a ``drive(path::PosixPath) = ""`` defind somewhere else.

Happy to change it, but shouldn't we change it for ``parts`` and ``root`` as well then?

Reply:
Oops, I forgot about the existing `drive(path::PosixPath) = ""` method. N/m that works then.